test/resources/demo-users/user/ns1/kustomization.yaml

```diff
@@ -6,4 +6,5 @@ resources:
   - application-and-component.yaml
   - ec-integration-test.yaml
   - release-plan.yaml
+  - afc.yaml
```

Comment:
Why do we need it to be in `user/ns1` ?

Reply:
We don't, it'll be removed when out of draft. It is meant for seeing the results in the CI